js/src/ui/Form/PasswordStrength/passwordStrength.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+import { debounce } from 'lodash';
+import { LinearProgress } from 'material-ui';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Would have expected at least the react-releated stuff to follow react, hence making exceptions jumbles it up and seems random. i.e. we typically do (not the case here)

```
import React from 'react';
import { FormattedMessage } from 'react-intl';
```

(grouped above, easy to see relations)

Here React is the exception, the rest sorted so it is a mix-match. Small thing (and driving me crazy), but like with everything, allows us to not have to think "in this case we do it differently".

Not a crisis, still worth being consistent about.